src/Components/test/E2ETest/Tests/RoutingTest.cs

```diff
@@ -109,6 +109,28 @@ public void CanArriveAtPageWithOptionalParametersNotProvided()
             Assert.Equal(expected, app.FindElement(By.Id("test-info")).Text);
         }
 
+        [Fact]
+        public void CanArriveAtPageWithCatchAllParameterWithEscapedSlashes()
+        {
+            SetUrlViaPushState("/WithCatchAllParameterWithEscapedSlashes/life/the/universe/and/everything = 42");
+
+            var app = Browser.MountTestComponent<TestRouter>();
+            var expected = $"The answer: life%2Fthe%2Funiverse%2Fand%2Feverything%20%3D%2042";
+
+            Assert.Equal(expected, app.FindElement(By.Id("test-info")).Text);
+        }
+
+        [Fact]
+        public void CanArriveAtPageWithCatchAllParameterWithoutEscapedSlashes()
+        {
+            SetUrlViaPushState("/WithCatchAllParameterWithoutEscapedSlashes/life/the/universe/and/everything = 42");
```

Comment:
I think this here might need to be in escaped syntax so that we can see the difference. I think that's why the tests are failing. On this one you should see the unescaped value, on the one above you should see the same value.

Reply:
Ah, ok, I did notice another call to `SetUrlViaPushState` had encoded values.  I'll get that changed.